src/hooks/useStaffSuppressInitialValue/useStaffSuppressInitialValue.js

```diff
@@ -0,0 +1,38 @@
+import {
+  useEffect,
+  useRef,
+} from 'react';
+import { useLocation } from 'react-router-dom';
+
+import { FACETS } from '../../constants';
+
+const useStaffSuppressInitialValue = (onChange, query) => {
+  const location = useLocation();
+  const isQueryFiltersSet = useRef(Boolean(query.filters));

```

Comment:
This can be simplified to `const isQueryFiltersSet = Boolean(query.filters);`. Therefore, some of the hooks can be removed.